benchmarks/bin/benchmarks.sh

```diff
@@ -20,7 +20,7 @@ for j in ../lib/*.jar; do
 done
 
 tso() {
-    exec java $JVM_FLAGS -Dlog4j.configuration=log4j.xml -cp $CLASSPATH org.apache.omid.benchmarks.tso.TSOServerBenchmark $@
+    exec java $JVM_FLAGS -Dlog4j.configuration=file:../conf/log4j.xml -cp $CLASSPATH org.apache.omid.benchmarks.tso.TSOServerBenchmark $@
```

Comment:
nope, it has to be like this according to the doc, should start with file:
If you refer to "../" part is was just plainly broken before.